public_suffix_list.dat

```diff
@@ -11204,6 +11172,80 @@ us-2.evennode.com
 // Submitted by Peter Ruibal <public-suffix@fb.com>
 apps.fbsbx.com
 
+// FAITID : https://faitid.org/ru/node/346
```

Comment:
// Foundation for Assistance for Internet Technologies and Infrastructure Development (FAITID) : https://faitid.org/en/projects/RU.NET/documents
// Submitted by Maxim Alzoba <@faitid.org>